src/config/musl.md

```diff
@@ -0,0 +1,67 @@
+# musl
+
+[musl](https://musl.libc.org/) is a libc implementation which strives to be
+lightweight, fast, simple, and correct.
+
+Void officially supports musl by using it in its codebase for all target
+platforms (although binary packages are not available for i686). Additionally,
+all compatible packages in our official repositories are available with
+musl-linked binaries in addition to their glibc counterparts.
+
+Currently, there are nonfree and debug subrepositories for musl, but no multilib
+subrepo.
+
+## Possible problems
+
+In some cases, software compatibility may be worse with musl.
```

Comment:
Something like this maybe?

Musl practices very strict and minimal standard compliance. Many commonly used platform specific extensions are not present. Because of this, it is common for software to need modification to compile and/or function properly. Void developers work to patch such software and hopefully get portability/correctness changes accepted into the upstream projects.

Reply:
Changed.